electron/src/preload/preload-webview.ts

```diff
@@ -243,14 +243,13 @@ process.once('loaded', () => {
     getDesktopSources: opts => ipcRenderer.invoke(EVENT_TYPE.ACTION.GET_DESKTOP_SOURCES, opts),
   };
   global.systemCrypto = {
-    decrypt: async (encrypted: Uint8Array): Promise<Uint8Array> => {
-      const plainText = await ipcRenderer.invoke(EVENT_TYPE.ACTION.DECRYPT, encrypted);
-      return Decoder.fromBase64(plainText).asBytes;
```

Comment:
this could cause an issue in case the value that was encrypted is not base64 encoded